velox/exec/HashTable.cpp

```diff
@@ -325,23 +354,31 @@ FOLLY_ALWAYS_INLINE void HashTable<ignoreNullKeys>::fullProbe(
 }
 
 namespace {
-// A normalized key is spread evenly over 64 bits. This mixes the bits
-// so that they affect the low 'bits', which are actually used for
-// indexing the hash table.
+// Normalized keys have non0-random bits. Bits need to be propagated
+// up to make a tag byte and down so that non-lowest bits of
+// normalized key affect the hash table index.
 inline uint64_t mixNormalizedKey(uint64_t k, uint8_t bits) {
```

Comment:
@oerling @Yuhta Can the bits parameter be deleted? Judging from the modification of the comments, this parameter seems to have been reserved on purpose, but not used.